templates/_topic/hsuforum_topic_button_meta.mustache

```diff
@@ -1,3 +1,3 @@
 <div class="last-reply-block">
-    <small class="hsuforum-thread-replies-meta">{{{lastposttime}}}<span class="pinned"><img src="pix/i/pinned.png" alt="pinned" /></span></small>
+    <small class="hsuforum-thread-replies-meta">{{{lastposttime}}}&nbsp;&nbsp;<span class="pinned rounded discussion-meta">{{pinned}}</span></small>
```

Comment:
Any reason for the double nbsp? Can the spacing not be achieved with css?

Reply:
would you prefer for it to be done in CSS?

Reply:
How much spacing do the nbsp add?

Reply:
2 spaces

Reply:
Keep it nbsp then. css feels a bit overkill